aggressive-fill-paragraph.el

```diff
@@ -59,11 +59,20 @@ by *, + or -."
        (or (eql (org-element-type (org-element-at-point)) 'table)
            (eql (org-element-type (org-element-at-point)) 'table-row))))
 
+(defun afp-in-org-src-block-header? ()
+  (interactive)
+  (let ((case-fold-search t))
+    (and (derived-mode-p 'org-mode)
+         (save-excursion
+           (beginning-of-line)
+           (looking-at-p "^[ \t]*#\\+\\(\\(begin\\|end\\)_src\\|header\\)")))))
```

Comment:
Does org-mode allow whitespace between any of the `#+header` etc? If so it might be good to allow that here too.

Reply:
If we allow that then we cannot insert any header args there. Here's an animated gif to demonstrate that:

![afp](https://user-images.githubusercontent.com/11584902/29995667-65afda3e-9021-11e7-8fe8-0a8046ab2ba9.gif)